README.md

```diff
@@ -188,17 +188,28 @@ git config blame.ignoreRevsFile .git-blame-ignore-revs
 
 On certain pages an API endpoint is called to receive experiment analysis data from Jetstream to display visualization tables. To see experiment visualization data, you must provide GCP credentials.
 
-1. Generate a GCP private key file.
+0. *prerequisite* Install GCP CLI
+    - Follow the instructions [here](https://cloud.google.com/sdk/docs/install)
+    - Project: `moz-fx-data-experiments`
 
-- Ask in #nimbus-dev for the GCP link to create a new key file.
-- Add Key > Create New Key > JSON > save this file.
-- Do not lose or share this file. It's unique to you and you'll only get it once.
+1. Authorize CLI with your account
+    - `gcloud auth login`
+    - `gcloud auth application-default login`
+      - this will save your credentials locally to a well-known location for use by any library that requests ADC
```

Comment:
Note:

I think if you run the docker setup before doing this step, because that filepath is referred to in a volume configuration, if none exists, Docker will create a stub in that paths place but of type directory, and then later when you try to run this authentication step it will fail to write:

```
ERROR: (gcloud.auth.application-default.login) Error saving Application Default Credentials: Unable to write file [/Users/jaredlockhart/.config/gcloud/application_default_credentials.json]: [Errno 21] Is a directory: '/Users/jaredlockhart/.config/gcloud/application_default_credentials.json'
jaredlockhart@Jareds-MacBook-Pro google-cloud-sdk % rm -Rf /Users/jaredlockhart/.config/gcloud/application_default_credentials.json
```

So we probably want to make a note to remove that stub dir if this happens.

Reply:
Ah yep, thanks! Added a note about this.